x-pack/plugins/enterprise_search/public/applications/enterprise_search/components/product_selector/product_selector.tsx

```diff
@@ -56,46 +56,48 @@ export const ProductSelector: React.FC<ProductSelectorProps> = ({ access }) => {
       <SendTelemetry action="viewed" metric="overview" />
 
       <EuiPageBody>
-        <TrialCallout />
-        <EuiPageHeader>
-          <EuiPageHeaderSection className="enterpriseSearchOverview__header">
-            <EuiTitle size="l">
-              <h1>
-                {i18n.translate('xpack.enterpriseSearch.overview.heading', {
-                  defaultMessage: 'Welcome to Elastic Enterprise Search',
-                })}
-              </h1>
-            </EuiTitle>
-            <EuiSpacer />
-            <EuiText>
-              <p>
-                {config.host
-                  ? i18n.translate('xpack.enterpriseSearch.overview.subheading', {
-                      defaultMessage: 'Select a product to get started.',
-                    })
-                  : i18n.translate('xpack.enterpriseSearch.overview.setupHeading', {
-                      defaultMessage: 'Choose a product to set up and get started.',
-                    })}
-              </p>
-            </EuiText>
-          </EuiPageHeaderSection>
-        </EuiPageHeader>
-        <EuiPageContentBody>
-          <EuiFlexGroup justifyContent="center" gutterSize="xl">
-            {shouldShowAppSearchCard && (
-              <EuiFlexItem grow={false} className="enterpriseSearchOverview__card">
-                <ProductCard product={APP_SEARCH_PLUGIN} image={AppSearchImage} />
-              </EuiFlexItem>
-            )}
-            {shouldShowWorkplaceSearchCard && (
-              <EuiFlexItem grow={false} className="enterpriseSearchOverview__card">
-                <ProductCard product={WORKPLACE_SEARCH_PLUGIN} image={WorkplaceSearchImage} />
-              </EuiFlexItem>
-            )}
-          </EuiFlexGroup>
-          <EuiSpacer size="xxl" />
-          {config.host ? <LicenseCallout /> : <SetupGuideCta />}
-        </EuiPageContentBody>
+        <main>
```

Comment:
This adds the `<main>` tag to reduce axe issues reported by axe-devtool and is primarily indenting changes, I recommend [viewing the diff with whitespace changes hidden](https://github.com/elastic/kibana/pull/108465/files?diff=split&w=1)

Reply:
Any chance you can upgrade this page to use `EuiPageTemplate`? It'll take care of the `main` tag for you and is the recommended approach of EUI.

Reply:
It's a fairly custom page in terms of design so I don't think it would be worth it (at least for this PR). IIRC from the newsletters, I believe there's a general overview design pass coming to several products across Kibana so this will likely get cleaned up then if that's cool!

Reply:
Looks like Caroline's converting our Overview to `KibanaPageTemplate` in https://github.com/elastic/kibana/pull/107709 (so fast!!) as well as removing the fancy background image, so I'll likely go ahead and remove this from my change from my PR completely. 🎉